include/xrpl/protocol/jss.h

```diff
@@ -164,6 +167,7 @@ JSS(attestations);
 JSS(attestation_reward_account);
 JSS(auction_slot);            // out: amm_info
 JSS(authorized);              // out: AccountLines
+JSS(authorize_credentials);   // in: ledger_entry DepositPreauth
```

Comment:
```suggestion
JSS(authorized_credentials);  // in: ledger_entry DepositPreauth
```

Reply:
It is from XLS for DepositPreauth - `AuthorizeCredentials`

Reply:
I'll update the spec - but IMO this should keep what is the norm in the RPC right now, which uses `authorized` in the RPC and `Authorize` in the transaction.

Reply:
Isn't it just a typo?

Reply:
I don't think so. It's `Authorize` in the transactions because that's the account/credentials you're actively authorizing. In the RPC it's `authorized` because those accounts/credentials have already been authorized.

Reply:
I agreed with the logic of creating names, that tx should be Authorize, and Ledger object should be Authorized. 

But ledger object now has fields `sfAuthorize` and `sfAuthorizeCredentials`.  And I think, that RPC, that return object by its fields values, should use exact the same parameter names as for object.  That's why I suspect that `authorized` is typo.   
So do you want to change just RPC `authorized_credentials` parameter, or also ledger object field to `AuthorizedCredentials`?

Reply:
I assume the ledger object uses the same so that you don't need to create two SFields.

I'd prefer matching like for like - so the RPC params match, and the transaction/ledger object params match.

Reply:
Now they differ only by notation - snake for RPC and pascal for ledger object and transactions

Reply:
No, I'm saying the RPC should be `authorized` and the objects/transactions should be `Authorize`, to match existing conventions.

Reply:
renamed to authorized_credentials